drivers/docker/utils_windows_test.go

```diff
@@ -0,0 +1,34 @@
+// +build windows
+
+package docker
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestExpandPath(t *testing.T) {
+	cases := []struct {
+		base     string
+		target   string
+		expected string
+	}{
+		{"/tmp/alloc/task", ".", "/tmp/alloc/task"},
+		{"/tmp/alloc/task", "..", "/tmp/alloc"},
+
+		{"/tmp/alloc/task", "d1/d2", "/tmp/alloc/task/d1/d2"},
+		{"/tmp/alloc/task", "../d1/d2", "/tmp/alloc/d1/d2"},
+		{"/tmp/alloc/task", "../../d1/d2", "/tmp/d1/d2"},
```

Comment:
Lots of duplication here.  Does it make to extract the common test cases here to a common test, or use by `GOOS` conditioning rather than conditional compilation?

Reply:
I figured it wasn't worth it for one fairly simple test - GOOS flagging on anonymous structs was pretty ugly, and adding a GOOS flag to the struct was p ugly